modules/administration/pages/custom-ssl.adoc

```diff
@@ -1,97 +1,289 @@
 [[custom-ssl]]
-= Using a Custom SSL Certificate
+= Custom SSL Certificates
 
+You can use custom certificates with {productname} and {productname} Proxy.
 
+This section covers how to use a third party SSL certificate authority with a new {productname} installation, and replacing existing certificate with new custom certificates.
 
+Before you begin, ensure you have:
 
+* A certificate authority (CA) SSL public certificate
+* An SSL server key
+* An SSL server certificate
 
+Your key and certificate files must be in PEM format.
 
-The following section will guide you through using a custom certificate with {productname} {productnumber} and {susemgrproxy} {productnumber}.
+The host name of the SSL keys and certificates must match the fully qualified host name of the machine you deploy them on.
+You can configure the host name of the certificate by editing the ``X509v3 Subject Alternative Name`` section of the certificate itself.
```

Comment:
```suggestion
You should set the host names in the ``X509v3 Subject Alternative Name`` section of the certificate.
```

"editing" sounds like you could use vi to change things, but certificates needs to be created using special tools.